openfisca_aotearoa/variables/social_security/69/community_services_card.py

```diff
@@ -97,14 +97,18 @@ class social_security__received_residential_care_subsidy(Variable):
     reference = """
         http://www.legislation.govt.nz/act/public/1964/0136/latest/DLM359124.html
```

Comment:
I would think we just need the link here and not repeat the description from the act?

Reply:
This line isn't changed in this PR - let's address later